azure-application-insights-spring-boot-starter/src/main/java/com/microsoft/applicationinsights/autoconfigure/ApplicationInsightsWebMvcAutoConfiguration.java

```diff
@@ -65,11 +73,40 @@ public FilterRegistrationBean webRequestTrackingFilterRegistrationBean(WebReques
         return registration;
     }
 
+    /**
+     * Programmatically registers an AppliactionInsightsServletContextListner to destroy all the running threads.
```

Comment:
Two typos in 'AppliactionInsightsServletContextListner'.
Happened to notice while learning the codebase :)